### DISCUSSION TITLE
Food Log

### DISCUSSION THREAD

**Discussion by BeatLink** [mealie-recipes/mealie#2146] [Feature Request]
**Before submitting this feature request I have**
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
There is no current way to track in mealie what was actually eaten on a given day a la food log / meal tracker / food journal

**(Optional): Suggest A Solution**
It would be great if mealie could implement an interface that allows you to track food. It could be presented as a calendar view, similar to the meal planner but would allow for adding individual food items and their quantities in addition to existing recipes. It would also be great if meals added to the meal planner could be quickly transferred to the food log

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
While not exactly what you're asking for, we recently implemented something similar in nightly

https://github.com/hay-kot/mealie/pull/1831

Honestly it would be pretty cool to put all these events on one master timeline, now that I think about it

**Comment by BeatLink** [comment]
This is a really cool feature! I can't wait to check it out. But would it be possible to have both a dedicated meal tracker using a calendar like interface in addition to the timeline for this and all other events?

**Comment by michael-genson** [comment]
There is now a global timeline that does this. Not exactly a calendar view, but it achieves the same thing: https://github.com/mealie-recipes/mealie/pull/2265
